apps/omg_status/lib/omg_status/metric/measurements.ex

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Status.Metric.Measurements do
```

Comment:
wait... why did we decide to remove `vmstats` and implement this on our own? you could have just specified a Statsd Sink and you'd be done...

Reply:
if we use the telemetry approach,  then we should use it everywhere?

Reply:
No :)